resources/dicts/relationship_events/normal_interactions/platonic/increase.json

```diff
@@ -484,8 +633,11 @@
 		"intensity": "high",
 		"interactions": [
 			"m_c thinks about how lucky {PRONOUN/m_c/subject} {VERB/m_c/are/is} to have someone like r_c as a friend.",
-			"m_c considers r_c to be one of {PRONOUN/m_c/poss} close friends.",
-			"m_c couldn't imagine how boring things would be without a friend like r_c."
+			"m_c considers r_c to be one of {PRONOUN/m_c/poss} closest friends.",
+			"m_c couldn't imagine how boring things would be without a friend like r_c.",
+			"m_c and r_c can't get enough of each others company.",
+            "m_c wonders if {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} r_c's friend in every reality...",
```

Comment:
can we have another thought that's: "m_c asks r_c if they're best friends in every universe" (though maybe put in in the platonic_60 event group since it's a lil bit of an escalation) solely because this one reminded me of that meme and that meme makes me emotional

Reply:
of course! I fully intended to have that but misremembered the quote so I'll add that one too lol

Reply:
Resolved